app/forms/admin/druid_search_form.rb

```diff
@@ -16,6 +16,8 @@ def work
     end
 
     def normalize_druid
+      druid.strip! # remove any trailing or leading whitespace the user may have inadvertantly entered via a copy/paste
```

Comment:
Typo: "inadvertently"

Reply:
your high school english teacher should be proud of this catch